src/parseValues.ts

```diff
@@ -68,3 +68,13 @@ export const parseValues = (values: any[], type: FieldType): any[] => {
       throw new Error('Unsupported field type');
   }
 };
+
+const parseStringAsTime = (timeString: string | null) => {
+  if (timeString === null) {
+    return timeString;
+  }
+  if (!isNaN(+timeString)) {
```

Comment:
The explicit convertion to number here is only to apease typescript, it's not needed in js.
We could add a tsignore comment instead.